Integration failures at v0.13.0

I'm getting integration failures on `orca`:

```
======================================================================
FAIL: test_dpr_curated_test_bf_bm25_hybrid_otf (test_dpr.TestSearchIntegration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/store/scratch/jimmylin/pyserini/integrations/dense/test_dpr.py", line 239, in test_dpr_curated_test_bf_bm25_hybrid_otf
    self.assertAlmostEqual(score, 0.9006, places=4)
AssertionError: 0.3314 != 0.9006 within 4 places

======================================================================
FAIL: test_dpr_curated_test_bf_otf (test_dpr.TestSearchIntegration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/store/scratch/jimmylin/pyserini/integrations/dense/test_dpr.py", line 215, in test_dpr_curated_test_bf_otf
    self.assertAlmostEqual(score, 0.8876, places=4)
AssertionError: 0.3285 != 0.8876 within 4 places

----------------------------------------------------------------------
```

--- a/docs/experiments-dpr.md
+++ b/docs/experiments-dpr.md
@@ -294,7 +294,8 @@ To evaluate, first convert the TREC output format to DPR's `json` format:
 $ python -m pyserini.eval.convert_trec_run_to_dpr_retrieval_run --topics dpr-curated-test \
                                                                 --index wikipedia-dpr \
                                                                 --input runs/run.dpr.curated-test.multi.bf.trec \
-                                                                --output runs/run.dpr.curated-test.multi.bf.json
+                                                                --output runs/run.dpr.curated-test.multi.bf.json \
+                                                                --regex
 
 $ python -m pyserini.eval.evaluate_dpr_retrieval --retrieval runs/run.dpr.curated-test.multi.bf.json --topk 20 100 --regex
 Top20  accuracy: 0.8876
@@ -315,7 +316,8 @@ To evaluate, first convert the TREC output format to DPR's `json` format:
 $ python -m pyserini.eval.convert_trec_run_to_dpr_retrieval_run --topics dpr-curated-test \
                                                                 --index wikipedia-dpr \
                                                                 --input runs/run.dpr.curated-test.bm25.trec \
-                                                                --output runs/run.dpr.curated-test.bm25.json
+                                                                --output runs/run.dpr.curated-test.bm25.json \
+                                                                --regex
 
 $ python -m pyserini.eval.evaluate_dpr_retrieval --retrieval runs/run.dpr.curated-test.bm25.json --topk 20 100 --regex
 Top20  accuracy: 0.8069
@@ -342,7 +344,8 @@ To evaluate, first convert the TREC output format to DPR's `json` format:
 $ python -m pyserini.eval.convert_trec_run_to_dpr_retrieval_run --topics dpr-curated-test \
                                                                 --index wikipedia-dpr \
                                                                 --input runs/run.dpr.curated-test.multi.bf.bm25.trec \
-                                                                --output runs/run.dpr.curated-test.multi.bf.bm25.json
+                                                                --output runs/run.dpr.curated-test.multi.bf.bm25.json \
+                                                                --regex
 
 $ python -m pyserini.eval.evaluate_dpr_retrieval --retrieval runs/run.dpr.curated-test.multi.bf.bm25.json --topk 20 100 --regex
 Top20  accuracy: 0.9006

